modules/stardate/tests/test_stardate.c

```diff
@@ -20,16 +20,21 @@
  *
  */
 
+#include <criterion/criterion.h>
+
 #include <syslog-ng.h>
 #include <logmsg/logmsg.h>
-#include <template_lib.h>
+#include <libtest/cr_template.h>
 #include <apphook.h>
 #include <plugin.h>
 
+<<<<<<< HEAD
```

Comment:
Missing resolve?

Reply:
Well.. Sorry, I forgot to press "save"..